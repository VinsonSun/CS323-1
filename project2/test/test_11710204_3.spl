int test_11710204_3()
{
   float a = 3.0;
   int a = 1;
   return a;
}
