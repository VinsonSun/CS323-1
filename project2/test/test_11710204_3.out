Error type 3 at Line 4: redefine variable: a
Error type 8 at Line 5: incompatiable return type
